### ISSUE TITLE
Look at axe workflow error

### ISSUE THREAD

**Issue by BorghildSelle** [equinor/energyvision#2618]
Look at last failing Test accessibility with axe-core on Browserstack github action workflow

**Comment by BorghildSelle** [comment]
Closing, as it works again
